packages/cli/src/aws/init.ts

```diff
@@ -372,7 +388,7 @@ async function checkOk(text: string): Promise<void> {
  * @param configFileName - The config file name.
  * @param config - The config file contents.
  */
-function writeConfig(configFileName: string, config: MedplumInfraConfig): void {
+function writeConfig(configFileName: string, config: Record<string, any>): void {
```

Comment:
```suggestion
function writeConfig(configFileName: string, config: Partial<MedplumInfraConfig>): void {
```
Can this be a `Partial`?